driver-service/src/main/java/com/vlad/kuzhyr/driverservice/utility/mapper/CarMapper.java

```diff
@@ -1,35 +1,32 @@
 package com.vlad.kuzhyr.driverservice.utility.mapper;
 
 import com.vlad.kuzhyr.driverservice.persistence.entity.Car;
-import com.vlad.kuzhyr.driverservice.persistence.entity.Driver;
-import com.vlad.kuzhyr.driverservice.web.request.CarRequest;
-import com.vlad.kuzhyr.driverservice.web.response.CarResponse;
+import com.vlad.kuzhyr.driverservice.web.dto.request.CarRequest;
+import com.vlad.kuzhyr.driverservice.web.dto.response.CarResponse;
 import org.mapstruct.BeanMapping;
 import org.mapstruct.InjectionStrategy;
 import org.mapstruct.Mapper;
 import org.mapstruct.Mapping;
 import org.mapstruct.MappingTarget;
-import org.mapstruct.Named;
 import org.mapstruct.NullValuePropertyMappingStrategy;
 
 @Mapper(componentModel = "spring",
```

Comment:
По возможности вместо строковых литералов следует использовать константы, особенно, если они уже определены в библиотеке.
```suggestion
@Mapper(componentModel = MappingConstants.ComponentModel.SPRING,
```